opencti-platform/opencti-graphql/src/utils/hub-client.ts

```diff
@@ -0,0 +1,37 @@
+import { getHttpClient } from './http-client';
+import conf, { logApp } from '../config/conf';
+
+type EnrollmentStatus = 'active' | 'inactive';
+
+const HUB_BACKEND_URL = conf.get('xtm:xtmhub_backend_url');
+
+export const hubClient = {
```

Comment:
I see that we don't have an "xtm hub" module yet, but we have an "xtm" one. Could you move this file to src/modules/xtm please ?